app/actions/todos.go

```diff
@@ -1,11 +1,27 @@
 package actions
 
 import (
+	"fmt"
 	"net/http"
+	"todos/app/models"
 
 	"github.com/gobuffalo/buffalo"
+	"github.com/gobuffalo/pop/v5"
 )
 
 func Index(c buffalo.Context) error {
+	tx := c.Value("tx").(*pop.Connection)
+	todos := models.Todos{}
+	err := tx.All(&todos)
+	if err != nil {
+		return err
```

Comment:
returning naked errors are hard to track when the app fails at this point.
You can use this instead, e.g
`c.Error(http.StatusInternalServerError, errors.Wrap(err, "Index - Error while getting all todos"))`
this same error management should be done for the rest of the app.

Reply:
Done!